src/Cello/cello.cpp

```diff
@@ -565,4 +565,37 @@ namespace cello {
   }
 
   //----------------------------------------------------------------------
+
+  bool is_initial_cycle(InitCycleKind kind) noexcept {
+    Simulation * sim = cello::simulation();
+    ASSERT("cello::is_first_cycle(InitCycleKind)",
+           "cello::simulation() returned nullptr", sim != nullptr);
+    return cello::is_initial_cycle(sim->cycle(), kind);
+  }
+
+  //----------------------------------------------------------------------
+
+  bool is_initial_cycle(int cycle, InitCycleKind kind) noexcept {
+    switch (kind) {
+    case InitCycleKind::fresh:
+      {
+        const Config * config = cello::config();
+        ASSERT("cello::is_initial_cycle(int, InitCycleKind)",
+               "cello::config() returned nullptr", config != nullptr);
+        return cycle == config->initial_cycle;
```

Comment:
Just wondering if there's a preferred way of accessing the `initial_cycle` variable (via `cello::config` or `cello::simulation` as in line 596). I'm not sure if there's a subtle difference between these but if not, maybe it's better to use the `simulation` interface here rather than accessing the `config` member directly?

In that case, using `fresh_or_noncharm_restart` inplace of `fresh` when calling these functions could save a little bit of code duplication.

Reply:
That's an excellent point!

It's been a long time since I wrote this. But, I vaguely recall that there is a difference in the 2 values based on whether or not we were doing a restart... (or I was concerned that the values might be different in the future). I'll look into that.

At the very least, I should document this with a comment.

Reply:
Just adding onto this. From what I can tell, `cello::simulation()->initial_cycle()` returns `cello::config()->initial_cycle`, so I don't think there is a difference between the two--though I haven't actually tested it to be sure.
You can see in `Simulation::initialize_simulation_()` where `cycle_initial_` is set to `config_->initial_cycle`.